.changeset/friendly-donkeys-sing.md

```diff
@@ -0,0 +1,8 @@
+---
+'@api3/airnode-deployer': minor
+'@api3/airnode-examples': minor
+'@api3/airnode-node': minor
+'@api3/airnode-validator': minor
+---
+
+Initialise packages for v0.4.0
```

Comment:
Don't we want to include all packages here if we're bumping the version?

Reply:
I dunno why github is showing the diff that badly... The PR is actually way smaller and most of the changes you reviewed were already merged.

Anyway, you are right - That PR which introduced only added validator templates so it's misleading.

Reply:
I'll double check the changelogs in a separate PR